src/Swarm/TUI/View.hs

```diff
@@ -317,7 +317,7 @@ drawGameUI s =
   addCursorPos = case s ^. uiState . uiWorldCursor of
     Nothing -> id
     Just coord ->
-      let worlCursorInfo = drawWorldCursorInfo (s ^. uiState . uiShowRobots) (s ^. gameState) coord
```

Comment:
Don't need this function argument anymore, because we won't hide the robot in the cursor display if the user pressed `Meta+h`.